include/mbedtls/ssl.h

```diff
@@ -470,14 +470,6 @@
 
 /** \} name SECTION: Module settings */
 
```

Comment:
Please remove the other deprecated things in `ssl.h` (alongside their implementation and tests):

- [ ] `MBEDTLS_SSL_MAJOR_VERSION_x`, `mbedtls_ssl_conf_max_version`, `mbedtls_ssl_conf_min_version`.
- [ ] `mbedtls_ssl_conf_sig_hashes`, `sig_hashes` field in `mbedtls_ssl_config`.